tests/func-tests/observability_controller_test.go

```diff
@@ -10,19 +10,33 @@ import (
 
 	v1 "k8s.io/api/core/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
 
+	"github.com/kubevirt/hyperconverged-cluster-operator/controllers/commontestutils"
 	"github.com/kubevirt/hyperconverged-cluster-operator/controllers/observability"
 	tests "github.com/kubevirt/hyperconverged-cluster-operator/tests/func-tests"
 )
 
 const testName = "observability_controller"
 
+var logger = logf.Log.WithName("observability-controller")
+
 var _ = Describe("Observability Controller", Label(tests.OpenshiftLabel, testName), func() {
+	var r *observability.Reconciler
```

Comment:
Not sure I understand this test. shouldn't the controller be part of the running HCO? why are we running it from here? this is a functional test, not unit test.